openfl-tutorials/interactive_api/HPU/PyTorch_TinyImageNet/envoy/requirements.txt

```diff
@@ -0,0 +1 @@
+Pillow==9.0.1
```

Comment:
This file can be removed if you have validated.

Reply:
This requirement.txt file in envoy folder is needed